integration-tests/tests/aws/kbv-check-happy.test.ts

```diff
@@ -0,0 +1,67 @@
+import { stackOutputs } from "./resources/cloudformation-helper";
+import { clearItems, populateTable } from "./resources/dynamodb-helper";
+import { executeStepFunction } from "./resources/stepfunction-helper";
+
+describe("HMRC KBV Check ", () => {
+  const stateMachineInput = {
+    sessionId: "12345",
+  };
+  const testUser = {
+    sessionId: "12345",
+    nino: "AA000003D",
+  };
+
+  const testQuestions = [
+    {
+      sessionId: "12345",
+      questionKey: "ita-bankaccount",
+      answered: "false",
+      correlationId: "93dcc67c-fe6d-4bd7-b68f-2bd848e0d572",
+      info: new Map(),
```

Comment:
Can you add a test for when there is information in the `info` section. At the moment you are using an empty map which is fine but we still need to test that the system doesn't break when there is and is not content in the `info` block as this is used as part of KBVs functional requirement.